tests/functional/pipelines/run_test.py

```diff
@@ -20,156 +20,143 @@
-)
-def test_run_with_url(batchmode: bool, version: str):
-    pipeline = PipelineFactory.list(query="SingleNodePipeline")["results"][0]
-
-    response = pipeline.run(
-        data="https://aixplain-platform-assets.s3.amazonaws.com/data/dev/64c81163f8bdcac7443c2dad/data/f8.txt",
-        **{"batchmode": batchmode, "version": version}
-    )
-    assert response["status"] == "SUCCESS"
-
-
-@pytest.mark.parametrize(
-    "batchmode,version",
-    [
-        (True, "2.0"),
-        (True, "3.0"),
-        (False, "2.0"),
-        (False, "3.0"),
-    ],
-)
-def test_run_with_dataset(batchmode: bool, version: str):
-    dataset = DatasetFactory.list(query="for_functional_tests")["results"][0]
-    data_asset_id = dataset.id
-    data_id = dataset.source_data["en"].id
-    pipeline = PipelineFactory.list(query="SingleNodePipeline")["results"][0]
-
-    response = pipeline.run(data=data_id, data_asset=data_asset_id, **{"batchmode": batchmode, "version": version})
-    assert response["status"] == "SUCCESS"
-
-
-@pytest.mark.parametrize(
-    "batchmode,version",
-    [
-        (True, "2.0"),
-        (True, "3.0"),
-        (False, "2.0"),
-        (False, "3.0"),
-    ],
-)
-def test_run_multipipe_with_strings(batchmode: bool, version: str):
-    pipeline = PipelineFactory.list(query="MultiInputPipeline")["results"][0]
-
-    response = pipeline.run(
-        data={"Input": "Translate this thing.", "Reference": "Traduza esta coisa."},
-        **{"batchmode": batchmode, "version": version}
-    )
-    assert response["status"] == "SUCCESS"
-
-
-@pytest.mark.parametrize(
-    "batchmode,version",
-    [
-        (True, "2.0"),
-        (True, "3.0"),
-        (False, "2.0"),
-        (False, "3.0"),
-    ],
-)
-def test_run_multipipe_with_datasets(batchmode: bool, version: str):
-    pipeline = PipelineFactory.list(query="MultiInputPipeline")["results"][0]
-
-    dataset = DatasetFactory.list(query="for_functional_tests")["results"][0]
-    data_asset_id = dataset.id
-
-    input_id = dataset.source_data["en"].id
-    reference_id = dataset.target_data["pt"][0].id
-
-    response = pipeline.run(
-        data={"Input": input_id, "Reference": reference_id},
-        data_asset={"Input": data_asset_id, "Reference": data_asset_id},
-        **{"batchmode": batchmode, "version": version}
-    )
-    assert response["status"] == "SUCCESS"
-
-
-@pytest.mark.parametrize("version", ["2.0", "3.0"])
-def test_run_segment_reconstruct(version: str):
-    pipeline = PipelineFactory.list(query="Segmentation/Reconstruction Functional Test - DO NOT DELETE")["results"][0]
-    response = pipeline.run("https://aixplain-platform-assets.s3.amazonaws.com/samples/en/CPAC1x2.wav", **{"version": version})
-
-    assert response["status"] == "SUCCESS"
-    output = response["data"][0]
-    assert output["label"] == "Output 1"
-
-
-@pytest.mark.parametrize("version", ["2.0", "3.0"])
+from aixplain.enums.asset_status import AssetStatus
+
+
+# def test_list_pipelines():
```

Comment:
All these tests are disabled somehow. Is that by purpose?

Reply:
The data is not being returned properly, so I commented some out to debug. Any idea why I could be getting success status, while no data is being returned?

Reply:
I have really no idea on that.